packages/connections/src/components/connections.spec.tsx

```diff
@@ -179,27 +197,41 @@ describe('Connections Component', function () {
   describe('connecting to a connection that is not succeeding', function () {
     let savedConnectableId: string;
     let savedUnconnectableId: string;
+    let tmpDir: string;
+
+    before(async function () {
+      tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'connections-tests'));
+      TestBackend.enable(tmpDir);
 
-    beforeEach(async function () {
       savedConnectableId = uuid();
       savedUnconnectableId = uuid();
       writeFakeConnection(tmpDir, {
         _id: savedConnectableId,
         port: 27018,
       });
-      writeFakeConnection(
-        tmpDir,
-        await convertConnectionInfoToModel({
-          id: savedUnconnectableId,
-          connectionOptions: {
-            // Hopefully nothing is running on this port.
-            // Times out in 5000ms.
-            connectionString:
-              'mongodb://localhost:28099/?connectTimeoutMS=5000&serverSelectionTimeoutMS=5000',
-          },
-        })
-      );
 
+      const connectionModel = await convertConnectionInfoToModel({
+        id: savedUnconnectableId,
+        connectionOptions: {
+          // Hopefully nothing is running on this port.
```

Comment:
we already use `get-port` to find available ports for the ssh tunnel, maybe we could use that here too, so we would be sure not to hit an used port, especially in evergreen